passwordless.go

```diff
@@ -136,16 +153,24 @@ func RequestToken(ctx context.Context, s TokenStore, t Strategy, uid, recipient
 	return nil
 }
 
-// VerifyToken checks the given token against the provided token store.
+// VerifyToken checks the given token against the provided token store, on successful
+// validation it deletes the token.
 func VerifyToken(ctx context.Context, s TokenStore, uid, token string) (bool, error) {
-	if isValid, err := s.Verify(ctx, token, uid); err != nil {
+	return VerifyTokenWithOptions(ctx, s, uid, token, WithValidDelete())
+}
+
+// VerifyTokenWithOptions
```

Comment:
Incomplete func doc comment here. It would be good to clearly describe how actions work, and what happens if an action returns an error, for example.